src/components/ImageView/index.native.js

```diff
@@ -69,6 +69,13 @@ class ImageView extends PureComponent {
                 imageWidth = Math.round(this.props.windowWidth);
                 imageHeight = Math.round(imageWidth * aspectRatio);
             }
+            const maxZoomRatio = 11;
+            if (width > (this.props.windowWidth * maxZoomRatio)) {
+                imageWidth = (this.props.windowWidth * maxZoomRatio);
+            }
+            if (height > (this.props.windowHeight * maxZoomRatio)) {
+                imageHeight = (this.props.windowHeight * maxZoomRatio);
+            }
```

Comment:
Let's add proper comments here. What is maxZoomRatio? Why do we need this?

Reply:
ok, I think the comment can be like this what do u think 
```ts 
const maxDimensionsScale = 11; // max image scale ration that  android can handle without crash 
```

Reply:
May be like 
```
// Resize the image to max dimensions possible on the Native platforms to prevent crashes on Android. To keep the same behavior, apply to IOS as well.
```